nrf-hal-common/Cargo.toml

```diff
@@ -44,6 +44,12 @@ version = "0.9.0"
 optional = true
 version = "0.9.0"
 
+
+[dependencies.nrf52833-pac]
+optional = true
+version = "0.9.0"
+git = "https://github.com/nrf-rs/nrf52833-pac.git"
```

Comment:
This should be published to crates.io before merge

Reply:
I guessed that was true, decided to get this out there to get comments.